factory/glideFactoryEntry.py

```diff
@@ -131,6 +131,11 @@ def __init__(self, name, startup_dir, glidein_descript, frontend_descript):
         )
         self.gflFactoryConfig.rrd_stats.base_dir = self.monitorDir
 
+        # Configure stale times
+        self.gflFactoryConfig.stale_maxage[1] = int(self.jobDescript.data["StaleAgeIdle"])
+        self.gflFactoryConfig.stale_maxage[2] = int(self.jobDescript.data["StaleAgeRunning"])
+        self.gflFactoryConfig.stale_maxage[-1] = int(self.jobDescript.data["StaleAgeUnclaimed"])
+
```

Comment:
You should add a try-except here, or even better a config validation, to make sure that the values are integers.